packages/template-retail-react-app/app/pages/account/addresses.jsx

```diff
@@ -321,47 +321,67 @@ const AccountAddresses = () => {
                         </>
                     )}
 
-                    {addresses.map((address) => (
-                        <React.Fragment key={address.addressId}>
-                            <ActionCard
-                                borderColor="gray.200"
-                                key={address.addressId}
-                                editBtnRef={editBtnRefs[address.addressId]}
-                                onRemove={() => removeAddress(address.addressId)}
-                                onEdit={() => toggleEdit(address)}
-                            >
-                                {address.preferred && (
-                                    <Badge
-                                        position="absolute"
-                                        fontSize="xs"
-                                        right={4}
-                                        variant="solid"
-                                        bg="gray.100"
-                                        color="gray.900"
-                                    >
-                                        <FormattedMessage
-                                            defaultMessage="Default"
-                                            id="account_addresses.badge.default"
-                                        />
-                                    </Badge>
-                                )}
-                                <AddressDisplay address={address} />
+                    {addresses.map((address) => {
+                        const editLabel = formatMessage(
+                            {
+                                defaultMessage: 'Edit {address}',
+                                id: 'account_addresses.label.edit_button'
+                            },
+                            {address: address.address1}
+                        )
+
+                        const removeLabel = formatMessage(
+                            {
+                                defaultMessage: 'Remove {address}',
+                                id: 'account_addresses.label.remove_button'
+                            },
+                            {address: address.address1}
+                        )
+
+                        return (
+                            <React.Fragment key={address.addressId}>
+                                <ActionCard
+                                    borderColor="gray.200"
+                                    key={address.addressId}
+                                    editBtnRef={editBtnRefs[address.addressId]}
+                                    onRemove={() => removeAddress(address.addressId)}
+                                    onEdit={() => toggleEdit(address)}
+                                    editBtnLabel={editLabel}
+                                    removeBtnLabel={removeLabel}
```

Comment:
There's another file that uses ActionCard: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/c9bc15e4ba6ea6447c13fdbc47098b1a25b46869/packages/template-retail-react-app/app/pages/checkout/partials/shipping-address-selection.jsx#L297

Should we update that file too? Or is that not in scope of the current ticket?

Reply:
@joeluong-sfcc We should fix it since it will come up for the next a11y edit. Would you mind creating a new PR to fix it?

Reply:
Sure thing, follow up PR here: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2041